### PR TITLE
Fixes for assertion failures in tests

### PR DESCRIPTION
**Pull request by atodorov**
This goes on top of #55.

### CODE REVIEW

**Comment by atodorov** [comment]
If I replace the verifyCancelButton() function call with its equivalent:

```diff
     test('clicking Cancel loads landing page', async () => {
         const [ , , cancel ] = verifyButtons();
-        verifyCancelButton(cancel);
+        cancel.click();
+
+        // this goes back to the landing page
+        await waitFor(
+            () => [
+                screen.getByText('Create a new image'),
+                screen.getByRole('button', { name: /Create image/ }),
+            ]
+        );
     });
```

then I get an assertion failure: https://travis-ci.com/github/osbuild/image-builder-frontend/builds/203753731 - debug output shows page hasn't been changed and the *Create image* button isn't available on the screen.

This tells me there's a problem with how `verifyCancelButton()` is currently implemented and that has been hiding an underlying problem which was discovered by the commented out assertion in #55.

**Comment by atodorov** [comment]
The following change leads to all assertions for the landing page failing:

```diff
- async function verifyCancelButton(cancel) {
+ function verifyCancelButton(cancel) {
    cancel.click();

    // this goes back to the landing page
-    await waitFor(
+    return waitFor(
        () => [
            screen.getByText('Create a new image'),
            screen.getByRole('button', { name: /Create image/ }),
@@ -100,7 +101,7 @@ describe('Step Release', () => {

    test('clicking Cancel loads landing page', async () => {
        const [ , , cancel ] = verifyButtons();
-        verifyCancelButton(cancel);
+        await verifyCancelButton(cancel);
    });
```

logs: https://travis-ci.com/github/osbuild/image-builder-frontend/builds/203755126

File: .eslintrc.yml [hunk 1]
```diff
@@ -86,7 +86,9 @@ rules:
     - error
     - functions: false
   no-undef: 2
-  no-unused-vars: 2
+  no-unused-vars:
+    - 2
+    - varsIgnorePattern: "^_"
```
Comment (Gundersanne) [inline .eslintrc.yml:91]: What does this do?

**Review by Gundersanne — APPROVED** [review]
Very nice! thank you

File: .eslintrc.yml (hunk 1 above)
Comment (atodorov) [reply .eslintrc.yml:91]: ignores unused variables whose name stars with underscore. Which ATM is used like this:
https://github.com/osbuild/image-builder-frontend/pull/56/files#diff-edcaaaf245bb6bcb9867fb52970fce66cdc1b83ca466911811c9ddb914212842R80

b/c I couldn't figure out how to assign only to the `history` variable ignoring everything else from the return value.

**Pull request merged**